examples/blog/routes/[slug].tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsx h */
+import { h } from "preact";
+import { tw } from "@twind";
+import { Handlers, PageProps } from "$fresh/server.ts";
+import Layout from "components/Layout.tsx";
+import { Marked } from "https://deno.land/x/markdown@v2.0.0/mod.ts";
+import { PostDetail } from "types/blog.ts";
+import { fetchRepoMetadata } from "utils/github.ts";
+import GithubRepo from "components/GithubRepo.tsx";
+import Tag from "../components/Tag.tsx";
+import BlogAuthor from "../components/BlogAuthor.tsx";
+
+export const handler: Handlers = {
+  async GET(req, ctx) {
+    try {
+      const BLOG_DIR = "blog";
+      const slug = ctx.params.slug;
```

Comment:
You have the option of using destructuring here to get rid of some redundancy:
`const { slug } = ctx.params`

Reply:
Updated